### PR TITLE
`Development`: Add exam test run e2e test

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Exams support test runs, where a instructor can test the exam before conducting it. 

### Description

This PR adds e2e tests for the test run within exams. We create a test run, change the working time, conduct it and delete it. 

#### Code Review
- [ ] Review 1
- [ ] Review 2

### CODE REVIEW

File: src/main/webapp/app/exam/manage/test-runs/test-run-management.component.html [hunk 1]
```diff
@@ -75,23 +75,23 @@ <h2><span jhiTranslate="artemisApp.examManagement.testRun.title">Test Run Manage
...
+                    <td class="testrun-id d-none d-md-table-cell">
                         <a [routerLink]="[testRun.id]">{{ testRun.id }}</a>
                     </td>
                     <td>
                         <a> {{ testRun.user?.name || '--' }}</a>
                     </td>
-                    <td>{{ testRun.workingTime || 0 | artemisDurationFromSeconds }}</td>
-                    <td>{{ (testRun.started ? 'global.generic.yes' : 'global.generic.no') | artemisTranslate }}</td>
-                    <td>{{ (testRun.submitted ? 'global.generic.yes' : 'global.generic.no') | artemisTranslate }}</td>
+                    <td class="working-time">{{ testRun.workingTime || 0 | artemisDurationFromSeconds }}</td>
```
Comment (krusche) [inline src/main/webapp/app/exam/manage/test-runs/test-run-management.component.html:85]: why do you use css classes here? I thought we want to consistently use ids?

Comment (TheZoker) [reply src/main/webapp/app/exam/manage/test-runs/test-run-management.component.html:85]: Since ids should only be used once per document, I decided to use classes here. But the table rows have a id attribute `testrun-{id}`, so within cypress I reference the elements with `#testrun-{id} .working-time`. A alternative could be to use ids like `testrun-{id}-working-time`. Personally I prefer the first approach, but I'm open for feedback!

**Review by pellumbbaboci — APPROVED** [review]
Code look good :+1:

File: src/test/cypress/support/constants.ts [hunk 2]
```diff
@@ -17,8 +17,8 @@ export const TIME_FORMAT = 'YYYY-MM-DDTHH:mm:ss.SSS';
 
 // Exercise types
 export enum EXERCISE_TYPE {
-    Text,
-    Modeling,
-    Quiz,
-    Programming,
+    Text = 'text',
```
Comment (rriyaldhi) [inline src/test/cypress/support/constants.ts:20]: Why are these changes necessary?

**Review by rriyaldhi — COMMENTED** [review]
Tests look good and are easy to understand 👍 . Just have 1 small question

File: src/test/cypress/support/constants.ts (hunk 2 above)
Comment (TheZoker) [reply src/test/cypress/support/constants.ts:20]: They are not relevant at the moment, but will be when I switch to using the exercise type enum from within the client code instead of building a separate enum:
https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/entities/exercise.model.ts#L33-L39

This will be done in a follow up. Thanks for the hin!
